dev/kyuubi-extension-spark-3-1/src/main/resources/sql-classification-default.json

```diff
@@ -1,34 +1,60 @@
 {
   "AlterDatabasePropertiesCommand": "ddl",
-  "AlterTableRenameCommand": "ddl",
-  "AlterTableRenamePartitionCommand": "ddl",
   "AlterTableAddColumnsCommand": "ddl",
-  "AlterTableChangeColumnCommand": "ddl",
   "AlterTableAddPartitionCommand": "ddl",
+  "AlterTableChangeColumnCommand": "ddl",
   "AlterTableDropPartitionCommand": "ddl",
-  "AlterTableSetPropertiesCommand": "ddl",
-  "AlterTableSetLocationCommand": "ddl",
-  "AlterTableSerDePropertiesCommand": "ddl",
-  "AlterTableUnsetPropertiesCommand": "ddl",
+  "AlterTableRecoverPartitionsCommand": "ddl",
```

Comment:
RepairTableCommand

Reply:
There is no such grammar.
"REPAIR TABLE" is belong to "AlterTableRecoverPartitionsCommand".